node/tests/swamp/swamp.go

```diff
@@ -296,3 +297,14 @@ func (s *Swamp) remove(rn *node.Node, sn []*node.Node) ([]*node.Node, error) {
 	}
 	return sn, nil
 }
+
+// Disconnect allows to break a connection between two peers without any possibility to re-establish it.
+// Order is very important here. We have to unlink peers first, and only after that call disconnect,
+// otherwise pubsub could re-establish the connection.
```

Comment:
Pubsub information is related to the place where the method is used, not on the Disconnect methods itself